src/Providers/ReinitializePayment/CashInAdvanceReinitializePaymentScript.php

```diff
@@ -4,14 +4,18 @@
 
 use Plenty\Plugin\Templates\Twig;
 use CashInAdvance\Helper\CashInAdvanceHelper;
+use CashInAdvance\Services\SessionStorageService;
 
 class CashInAdvanceReinitializePaymentScript
 {
-	public function call(Twig $twig): string
+	public function call(Twig $twig, SessionStorageService $sessionStorageService): string
```

Comment:
We should you use the `Plenty\Modules\Webshop\Contracts\LocalizationRepositoryContract` here.

Reply:
Habe es geändert. Der `SessionStorageService` wird aber noch an anderen Stellen in dem Plugin für die Sprache verwendet.

Reply:
Ok, than this should be replaced step by step. The LocalizationRepositoryContract is a new Class from the Webshop Module. We also use this class in the Klarna, Mollie and PayPal Plugin.